controllers/cohortdefinition.go

```diff
@@ -70,13 +73,48 @@ func (u CohortDefinitionController) RetriveStatsBySourceIdAndTeamProject(c *gin.
 	if err1 == nil {
 		cohortDefinitionsAndStats, err := u.cohortDefinitionModel.GetAllCohortDefinitionsAndStatsOrderBySizeDesc(sourceId, teamProject)
 		if err != nil {
-			c.JSON(http.StatusInternalServerError, gin.H{"message": "Error retrieving cohortDefinition", "error": err.Error()})
+			c.JSON(http.StatusInternalServerError, gin.H{"message": "Error retrieving cohortDefinitions for 'team project' role", "error": err.Error()})
```

Comment:
Is it possible to use actual teamProject name instead of 'team project'

Reply:
we could, but in this case it would not add any useful information, and might actually cause more confusion. This part is really about reporting an InternalServerError and the message details help us to find where it happened. These errors are most likely to be a DB timeout, or some other underlying infra issue, and not really related to any value in the teamProject variable (and if it is, hopefully that would become evident in the `err.Error()` message. 
Added to this, I do prefer not to have too many places where the user input is echoed back to the output or logged.